app/views/distributions/edit.html.erb

```diff
@@ -17,6 +17,14 @@
 <!-- Main content -->
 <section class="content">
 
+<% if @distribution.issued_at < Date.today %>
+    <div class="alert alert-warning", role="alert">
+      The current date is past the date this distribution was picked up.
+      Please be very careful when editing this record;
+      the contents should match what was given to the recipient.
+    </div>
+<% end %>
+
```

Comment:
This is not a blocker, but if you would like, feel free to add a `past?` method (or similar) to `Distribution` that does that inequality you have there -- looks like we're using it in quite a few places!